### PR TITLE
Add metrics validation to avoid NaN #253

### PR DESCRIPTION
**Pull request by ivarconr**
Solves #253

### CODE REVIEW

**Comment by sveisvei** [comment]
👍

File: lib/client-metrics/index.js [hunk 1]
```diff
@@ -91,6 +91,12 @@ module.exports = class UnleashClientMetrics {
         return this.apps[appName];
     }
 
+    createCountObject(entry) {
+        const yes = typeof entry.yes == 'number' ? entry.yes : 0;
+        const no = typeof entry.no == 'number' ? entry.no : 0;
```
Comment (ivarconr) [inline lib/client-metrics/index.js]: use `====`

Comment (sveisvei) [inline lib/client-metrics/index.js]: Indeed - eslint rules should have catched this ... check eslint stuff?

Comment (SimenB) [inline lib/client-metrics/index.js]: We have "smart" mode activated for `eqeqeq`, and it allows `typeof` to be double equals as there is no chance of type coercian.

http://eslint.org/docs/rules/eqeqeq#smart

`node_modules/.bin/eslint --print-config lib/client-metrics/index.js | jq  '.rules.eqeqeq'`

File: lib/client-metrics/client-metrics.test.js [hunk 2]
```diff
@@ -335,3 +335,45 @@ test('should have correct values for lastHour', t => {
...
+            toggles: {
+                toggleX: {
+                    blue: 10,
+                    green: 10,
+                },
+            },
+        },
+    });
+
+    t.truthy(metrics.globalCount === 123);
```
Comment (SimenB) [inline lib/client-metrics/client-metrics.test.js]: `t.true(metrics.globalCount === 123)` or `t.is(metrics.globalCount, 123)`

**Pull request merged**